### PR TITLE
ci: skip some find element test cases for api-25 as they are flaky

### PR DESCRIPTION
**Pull request by rerorero**
I noticed functional test for api-25 always fails on the some specific test cases
e.g.
https://github.com/appium/appium-uiautomator2-driver/actions/runs/4752045785/jobs/8441915286?pr=597
https://github.com/appium/appium-uiautomator2-driver/actions/runs/4744576967/jobs/8425670890?pr=596
https://github.com/appium/appium-uiautomator2-driver/actions/runs/4729447613/jobs/8391973545
https://github.com/appium/appium-uiautomator2-driver/actions/runs/4608686473/jobs/8144798439

I can't reproduce it on my laptop then it'd take time to investigate.
Could we just skip them for now?

### CODE REVIEW

**Comment by rerorero** [comment]
now another tests started failing 😢 
https://github.com/appium/appium-uiautomator2-driver/actions/runs/4752784058/jobs/8443556898?pr=598

**Comment by rerorero** [comment]
I ended up with skipping all `find` tests for api-25

File: .github/workflows/functional-test.yml [hunk 1]
```diff
@@ -64,10 +64,11 @@ jobs:
     - uses: reactivecircus/android-emulator-runner@v2
       name: e2e_api${{ matrix.apiLevel }}
       with:
+        # skip find tests with api-25 as they are too flaky
         script: |
           npm run e2e-test:driver
           npm run e2e-test:commands
-          npm run e2e-test:commands:find
+          [[ "${{ matrix.apiLevel }}" == "25" ]] || npm run e2e-test:commands:find
```
Comment (mykola-mokhnach) [inline .github/workflows/functional-test.yml:71]: Could you please add a comment here why this is needed?

Comment (mykola-mokhnach) [reply .github/workflows/functional-test.yml:71]: A link to this Pr would also be ok

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by rerorero** [comment]
I skipped more tests since it was still pretty unstable.

**Comment by mykola-mokhnach** [comment]
I was thinking about maybe excluding API 25 completely and adding API 31/32 instead.
We could keep only the necessary minimum for API 23 though, kind'a smoke check

**Pull request merged**